src/vector_math.h

```diff
@@ -8,10 +8,15 @@
 
 #define ks_lt_index(a, b) ((a).value < (b).value)
 
-#if   defined(USE_SSE)
-#include <emmintrin.h>
+#if defined(USE_SSE)
+	#if defined(USE_NEON)
+		#include "sse2neon.h"	
```

Comment:
Is this change related to the PR?

Reply:
No sorry, that was just another change for a project I'm working, I can create a new branch for this PR and re-open it

Reply:
Re-opened here https://github.com/openvenues/libpostal/pull/669